### ISSUE TITLE
How to remove invalid proxy-status item（Virtual Machine）?

### ISSUE THREAD

**Issue by cfanbo** [istio/istio#34775]
**description**
install istio on vm, but found many invalid proxy-status item
```
$istioctl proxy-status
vm1.vm                                                  SYNCED       SYNCED       SYNCED       SYNCED       istiod-568d797f55-g6gmv     1.10.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  SYNCED       SYNCED       SYNCED       SYNCED       istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
vm2.vm                                                  NOT SENT     NOT SENT     NOT SENT     NOT SENT     istiod-568d797f55-g6gmv     1.11.0
```

[ ] Docs
[X] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure
[ ] Upgrade

**Expected behavior**
only show vaild proxy-status item

**Steps to reproduce the bug**

Due to configuration error, the virtual machine has failed to connect multiple times

**Version** (include the output of `istioctl version --remote` and `kubectl version --short` and `helm version --short` if you used Helm)

**How was Istio installed?**

https://istio.io/latest/docs/setup/install/virtual-machine/

**Environment where the bug was observed (cloud vendor, OS, etc)**
Ubuntu 18.04.5 LTS

Additionally, please consider running `istioctl bug-report` and attach the generated cluster-state tarball to this issue.
Refer [cluster state archive](http://istio.io/help/bugs/#generating-a-cluster-state-archive) for more details.

**Comment by cfanbo** [comment]
now haveTwo virtual machine
```
root@sxf-virtual-machine:~/data/mtls# kubectl get we -n vm
NAME                  AGE   ADDRESS
devapp-192.168.3.62   26h   192.168.3.62
myapp-192.168.3.56    26h   192.168.3.56
```

**Comment by howardjohn** [comment]
sounds a lot like https://github.com/istio/istio/pull/31899 but that should already be fixed in 1.11. What patch version are you on?
